### DISCUSSION TITLE
11.0 release

### DISCUSSION THREAD

**Discussion by joaopaulocruz** [markusaksli/TradeBot#36] [Q&A]
@markusaksli thanks for your project, it's a very nice project.

Can you tell us when you think to finish the 11.0 release?

Thanks

**Comment by markusaksli** [comment]
Thanks for the kind words

I can't give a good estimate because I'm the only one working on this project but I'm also finishing my bachelor's, full-time employed, and doing game dev. It really bothers me that v11 isn't out yet and it's nagging on me all the time but I just haven't found the time to work on it during the semester yet.

The project isn't abandoned, but I doubt v11 and the front-end Flutter app will be out in all their glory before next summer. I might cut the SSL functionality from it and just focus on reimplementing a CLI for the new instance system so it can be a basic working version but even then I can't promise any estimate.
